src/components/paragraph/staff_profile/index.tsx

```diff
@@ -0,0 +1,111 @@
+import Image from '@/components/image'
+import { DEV_PATH } from '@/lib/constants'
+
+export const StaffProfiles = ({ node }): JSX.Element => {
```

Comment:
So, a slightly tricky thing here is that, if we're rendering out a staff_profile _paragraph_, what is actually going to get passed to this component is paragraph data. The node that actually contains the person's name, biography, phone number, etc, is attached to a field on that paragraph.

The existing liquid template deals with this in a sort of simple way, by drilling into that field value to get the attached node and assigning it to a variable:
https://github.com/department-of-veterans-affairs/content-build/blob/master/src/site/paragraphs/staff_profile.drupal.liquid#L1 

We could do something similar here: 
```
export const StaffProfiles = ({ paragraph }): JSX.Element => {
  const node = paragraph.field_staff_profile;
  if (!node) return;
  <snip>
```

Then the rest of the component operates as-is.